### DISCUSSION TITLE
Gallery page?

### DISCUSSION THREAD

**Discussion by hvddrift** [McShelby/hugo-theme-relearn#653] [Q&A]
Hi

This theme is fantastic. Going to use it as a support site for our product. On my current site (so old looks like a MySpace page) I had a gallery page of completed installations. As a quite new user to Hugo and relearn other than a series of Markdown images can you make a gallery page like the ones I see around using HUGO? When I try I seem to run into issues.

**Comment by McShelby** [comment]
While I see some sense and advantages having a gallery feature, this is nothing that is planned in the near future.

As there are so many different implementations, could you give some examples how you would expect a gallery to look like?

**Comment by hvddrift** [comment]
I watched this video: https://youtu.be/s61ZwPC4Q7g?si=Q_iqTbvWp5fdtMsE

and it looked interesting

and his GitHub : https://github.com/future-wd/video-hugo-instagram-gallery
